### DISCUSSION TITLE
Branding: PowerShell or PowerShell Core?

### DISCUSSION THREAD

**Discussion by God-damnit-all** [PowerShell/PowerShell#16386] [Q&A]
PowerShell 7 seemed to ditch the PowerShell Core branding, but honestly, that's not been to its benefit. Saying PowerShell v6+ seems awkward when PowerShell is at v7 and soon going on v8.

However, README.md still refers to the project as PowerShell Core, so I'm just a bit uncertain as to how the project wants to brand itself going forward. I really hope the 'PowerShell Core' moniker sticks around, personally.

**Comment by doctordns** [comment]
The decision to drop the "core" bit was, IIRC, made at the same time the .NET team did the same. Going forward, it's just PowerShell (and just .NET). The core moniker is gone and I suspect it will not make a return. 

FWIW: PowerShell 8 is not, I think, something on the roadmap. 7.2 is the next release (very soon now) and I would expect the release after that to be 7.3. 

That said, the readme document could usefully be updated.
